core/src/avm2/globals/flash/media/Sound.as

```diff
@@ -0,0 +1,27 @@
+package flash.media {
+    import flash.events.EventDispatcher;
+    import flash.utils.ByteArray;
+    import flash.net.URLRequest;
+    
+    [Ruffle(InstanceAllocator)]
+    public class Sound extends EventDispatcher {
+        public function Sound(stream:URLRequest = null, context:SoundLoaderContext = null) {
+            this.init(stream, context)
+        }
+        private native function init(stream:URLRequest, context:SoundLoaderContext);
+
+
+        public native function get bytesLoaded():int;
```

Comment:
`bytesLoaded` is actually a `uint`. [AS3 Ref](https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/media/Sound.html#bytesLoaded)